public/app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+
+import { dateTimeFormat } from '@grafana/data';
+import { Modal, useStyles2, Button } from '@grafana/ui';
+import { PMM_UPDATES_LINK } from 'app/percona/shared/components/PerconaBootstrapper/PerconaNavigation';
+import { PerconaUpdateVersionProps } from 'app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.types';
+import { checkUpdatesChangeLogs, UpdatesChangeLogs } from 'app/percona/shared/core/reducers/updates';
+import { setSnoozedVersion } from 'app/percona/shared/core/reducers/user/user';
+import { getPerconaUser, getUpdatesInfo } from 'app/percona/shared/core/selectors';
+import { useAppDispatch } from 'app/store/store';
+import { useSelector } from 'app/types';
+
+import { Messages } from './PerconaUpdateVersion.constants';
+import { getStyles } from './PerconaUpdateVersion.styles';
+
+const PerconaUpdateVersion = ({ showUpdate, setShowUpdate }: PerconaUpdateVersionProps) => {
```

Comment:
I think you can move the state to the updates store, then you could access the show modal state anywhere in the tree.

Reply:
fixed